src/Symfony/Component/Serializer/SerializerAwareTrait.php

```diff
@@ -12,6 +12,8 @@
 namespace Symfony\Component\Serializer;
 
 /**
+ * @deprecated Use NormalizerAwareTrait instead
```

Comment:
I won't suggest any replacement as the normalizer doesn't fulfill all the serializer features

Reply:
True... But all the usages of `SerializerAwareTrait` in core has been used for normalizing. I'll remove it.